Source/WebKit/NetworkProcess/PrivateClickMeasurement/PrivateClickMeasurementClientImpl.cpp

```diff
@@ -45,16 +45,17 @@ void ClientImpl::broadcastConsoleMessage(JSC::MessageLevel messageLevel, const S
 
 bool ClientImpl::featureEnabled() const
 {
-    return m_networkSession
-        && m_networkProcess->privateClickMeasurementEnabled()
-        && !m_networkSession->sessionID().isEphemeral();
+    return m_networkSession && m_networkProcess->privateClickMeasurementEnabled();
```

Comment:
I remember Brady Eidson making use of how this was tied to ephemeral sessions for some unrelated things. Please check with him on what that was. It may have been regarding the special stateless NSURLSession PCM uses for sending attribution reports and how it relates to ephemeral sessions. It may also be the case that that issue is no longer relevant after the introduction of the daemon.

Reply:
Interesting! I'll double check that — my understanding is that `PCM::NetworkLoader` already uses an ephemeral `NSURLSession` for reporting attribution (which should be compatible with our browsing session), but perhaps there's a subtlety here that I'm missing.